src/nvim/popupmenu.c

```diff
@@ -878,7 +863,7 @@ static void pum_adjust_info_position(win_T *wp, int width)
   wp->w_config.anchor = pum_above ? kFloatAnchorSouth : 0;
   linenr_T count = wp->w_buffer->b_ml.ml_line_count;
   wp->w_width_inner = wp->w_config.width;
-  wp->w_config.height = MIN(Rows, pum_preview_win_height(wp, count));
+  wp->w_config.height = plines_m_win(wp, 1, count, Rows);
```

Comment:
Is the topline always 1 in the preview window? Or should this be:
```suggestion
  wp->w_config.height = plines_m_win(wp, wp->w_topline, count, Rows);
```

Reply:
emm there can bind the scroll key to scroll it

Reply:
Question is if that should affect the height of the window. I guess not in which case it's fine.

Reply:
I have updated comment :)